### DISCUSSION TITLE
How to use clerk react native without expo?

### DISCUSSION THREAD

**Discussion by rafakwolf** [clerk/javascript#2259] [Support request - DO NOT OPEN HERE]
Hi folks ...

I am trying to use clerk in a react-native-cli app, but I am facing some errors, like saying I don't have the expo-auth-session installed, and then saying I don't have expo-module-core installed. So I installed those packages, but then this error appears:
```
Error: Cannot find native module 'ExpoWebBrowser', js engine: hermes
```

Even I have expo-we-browser installed as well, can I have some help?

Thank you so much!

**Comment by LucasReinaldo** [comment]
hey @LekoArts, is there any chance this is related to your area of expertise?

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
